### ISSUE TITLE
[BUG] Missing ability to go back to Fullscreen Video

### ISSUE THREAD

**Issue by GalacticWave** [jurialmunkey/skin.arctic.horizon.2#887]
### Skin section

Fullscreen Video OSD

### Current Behavior

When playing a video, if back button is pressed on the remote, the movie/episode list comes up. From here there is no way to go back to the fullscreen video that is playing now. 
![68D35135-9AD5-417F-84E1-E85250B7292E](https://user-images.githubusercontent.com/77709470/227741490-1cbbb579-8bfa-4ce1-9b13-dbe768106cc0.jpeg)

In Arctic Horizon version 1, there was an option to go back clicking on Fullscreen option. I don't have version 1 installed anymore and therefore couldn't take a screenshot but here is the screenshot from the default Estuary skin. It's the highlighted fullscreen icon. 
![4AE10C96-54CE-4D3B-9F88-9D8EAB6EC535](https://user-images.githubusercontent.com/77709470/227741608-90d4918d-2bd9-410d-ad17-c568404f4887.jpeg)

Would be great if that fullscreen option can be put back in the options menu when a video is playing in the background. 

### Expected Behavior

While playing a video, if back button is clicked which loads the list screen while playing the video in the background, there should be an option to go back to Fullscreen video in the options menu. 

### Steps To Reproduce

1. Play a video
2. Click back button on the remote
3. Video playlist screen comes up
4. Pressing ip button the remote shows the options menu
5. But the options menu is missing the fullscreen option to go back to Fullscreen video

### Checklist

- [x] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
It's there for me

![Screenshot 2023-03-26 134700](https://user-images.githubusercontent.com/4618719/227752468-b7afb307-0586-4bfb-9090-25f50689038f.jpg)

**Comment by GalacticWave** [comment]
Strange. Is there a setting to have that option? I have the latest Android TV version 10, Kodi 20.1, and AH2 v0.8.17. Uninstalled and reinstalled couple of times and still not getting those options. I will install on windows os and check it out. I will also try on a fresh install of Kodi 20.1. 

BTW, here is the video link showing the problem. 
https://drive.google.com/file/d/16Ze7NJNQCl7zRMxWUPsLQnLbaMQsMni4/view?usp=drivesdk

**Comment by GalacticWave** [comment]
A quick update: 
I just did a fresh start of Kodi 20.1 on the Sony A80J Google TV (Android 10) and only had the AH2 skin installed. Still not getting the option. 
Next:
I will install on my windows 11 computer with just the  skin and clean 20.1 install and check how that goes.
Note:
I had AH1 working fine on this same TV. It broke after AH2 was installed.

**Comment by GalacticWave** [comment]
Found the difference. The options shows up when a library folder is added as a video source. After installing AH2, I didn't do that. Was running videos directly from addons.

Question: Kodi's default skin Estuary has the options available without Library folder attached. Can this be done for AH2 as well? If not, it's not a big deal as I do have a library of owned videos but might be helpful for users who only play videos from addons like YouTube or IMDB Trailers.

**Comment by jurialmunkey** [comment]
The options show when the playlist queue has 1 or more items in it (ie something is playing).

This should also work for addons (works for me off things like sly guy plugins). It sounds like the addons you're using are starting playback via player hacks rather than using the proper setresolvedurl method.

**Comment by admiral-aqua** [comment]
Don't see that menu option either. Neither with a fresh profile and skin install nor with standard profile. Tested both a local file and a streaming addon. The only workaround I found, is simply pressing back when I am in the home menu, which brings me back to the video, which is almost even better since it's one menu less to hassle with.

**Comment by GalacticWave** [comment]
> This should also work for addons (works for me off things like sly guy plugins). It sounds like the addons you're using are starting playback via player hacks rather than using the proper setresolvedurl method.

I am not sure if it's an addon issue. Because I used multiple addons including the youtube addon. When I switch to Estuary, the fullscreen option is there but when I switched back to AH2, the options were not there. When I added a video source to my library, all the options came back. 

But on a fresh install with no library and only youtube plugin, the options are not there even when playing a video.
